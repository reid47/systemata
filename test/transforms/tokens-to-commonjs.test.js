const { generateTokenTransformTests } = require('../test-utils');

generateTokenTransformTests({ transformName: 'tokensToCommonJs' });
