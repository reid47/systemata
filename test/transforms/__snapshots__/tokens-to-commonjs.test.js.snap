// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokensToCommonJs simple case 1`] = `
"module.exports = {
  colorPrimary: '#f00',

  colorSecondary: '#0f0',

  colorTertiary: '#00f',

  spacingSm: '8px',

  spacingMd: '16px',

  spacingLg: '24px'
};"
`;

exports[`tokensToCommonJs with a namespace 1`] = `
"module.exports = {
  rmColorPrimary: '#f00',

  rmColorSecondary: '#0f0',

  rmColorTertiary: '#00f',

  rmSpacingSm: '8px',

  rmSpacingMd: '16px',

  rmSpacingLg: '24px'
};"
`;

exports[`tokensToCommonJs with comments 1`] = `
"module.exports = {
  // This is the primary color.
  colorPrimary: '#f00',

  // This is the secondary color.
  // It should be used in some places.
  colorSecondary: '#0f0',

  colorTertiary: '#00f',

  spacingSm: '8px',

  spacingMd: '16px',

  spacingLg: '24px'
};"
`;
